templates/login.html

```diff
@@ -1,312 +1,314 @@
+                                </div>
+                                <div class="form-group">
+                                  <input type="password" name="password" class="form-control" placeholder="Password" id="psw" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}" placeholder="Password" title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters" required>
+                                    <div class="errorarea1" id="errorarea1">
+                                      <div id="message" class="col col-11">
+                                        <p id="letter" class="invalid">A <b>lowercase</b> letter</p>
+                                        <p id="capital" class="invalid">A <b>capital (uppercase)</b> letter</p>
+                                        <p id="number" class="invalid">A <b>number</b></p>
+                                        <p id="length" class="invalid">Minimum <b>8 characters</b></p>
+                                      </div>
+                                    </div>
+                                    {{errorP}}
+                                </div>
+                                <div class="form-group">
+                                  <input type="password" name="rpassword" class="form-control" placeholder="Confirm Password" id="psw2" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}" title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters" required>
+                                        <div class="errorarea2" id="errorarea2">
+                                        <div id="message2" class="col col-11">
+                                        <p id="passwordmatch" class="invalid"> <b>Password</b>Matched</p>
+                                    </div>
+                                        </div>
+                                  {{errorPC}}
+                                </div>
+
+                                <div class="form-group">
+                                    <div class="g-recaptcha" data-sitekey="6LfEWpIaAAAAAF-fgAPji1I65tIdQNERnTgYF0Yg"></div>
+                                </div>
+                                <div class="form-group">
+                                    {% if messages %}
+                                    <div>
+                                      {% for message in messages %}
+                                      {% if message.tags == 'error' %}
+                                      <div class="alert alert-danger" style="margin-top:10px">{{ message }}</div>
+                                      {% endif %}
+                                      {% endfor %}
+                                    </div>
+                                    {% endif %}
+                                </div>
+                                <i class="ion-ios-arrow-right"></i> <a href="{% url 'forgotp' %}">Forgot Password?</a>
```

Comment:
Can u pl. remove this line from signup form

Reply:
169 or 170 pl check

Reply:
done sir